src/app/hooks/useDeterminePlaceholderMode/index.tsx

```diff
@@ -0,0 +1,87 @@
+import useOperaMiniDetection from '#app/hooks/useOperaMiniDetection';
+import { Services } from '#app/models/types/global';
+import { useContext, useEffect, useState } from 'react';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+// Disabled due to bug in ts lint
+// eslint-disable-next-line no-shadow
+export enum Mode {
```

Comment:
Should we rename this?
```suggestion
export enum PlaceholderMode {
```